undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemParser_10_0.java

```diff
@@ -0,0 +1,441 @@
+                                                        ReverseProxyHandler.PROBLEM_SERVER_RETRY,
+                                                        ReverseProxyHandler.MAX_REQUEST_TIME,
+                                                        ReverseProxyHandler.REQUEST_QUEUE_SIZE,
+                                                        ReverseProxyHandler.CACHED_CONNECTIONS_PER_THREAD,
+                                                        ReverseProxyHandler.CONNECTION_IDLE_TIMEOUT,
+                                                        ReverseProxyHandler.MAX_RETRIES)
+                                                .addChild(builder(ReverseProxyHandlerHost.INSTANCE.getPathElement())
+                                                        .setXmlElementName(Constants.HOST)
+                                                        .addAttributes(
+                                                                ReverseProxyHandlerHost.OUTBOUND_SOCKET_BINDING,
+                                                                ReverseProxyHandlerHost.SCHEME,
+                                                                ReverseProxyHandlerHost.PATH,
+                                                                ReverseProxyHandlerHost.INSTANCE_ID,
+                                                                ReverseProxyHandlerHost.SSL_CONTEXT,
+                                                                ReverseProxyHandlerHost.SECURITY_REALM,
+                                                                ReverseProxyHandlerHost.ENABLE_HTTP2))
+                                )
+
+
+                )
+                .addChild(
+                        builder(FilterDefinitions.INSTANCE.getPathElement())
+                                .setXmlElementName(Constants.FILTERS)
+                                .setNoAddOperation(true)
+                                .addChild(
+                                        builder(RequestLimitHandler.INSTANCE.getPathElement())
+                                                .addAttributes(RequestLimitHandler.MAX_CONCURRENT_REQUESTS, RequestLimitHandler.QUEUE_SIZE)
+                                ).addChild(
+                                builder(ResponseHeaderFilter.INSTANCE.getPathElement())
+                                        .addAttributes(ResponseHeaderFilter.NAME, ResponseHeaderFilter.VALUE)
+                        ).addChild(
+                                builder(GzipFilter.INSTANCE.getPathElement())
+                        ).addChild(
+                                builder(ErrorPageDefinition.INSTANCE.getPathElement())
+                                        .addAttributes(ErrorPageDefinition.CODE, ErrorPageDefinition.PATH)
+                        ).addChild(
+                                builder(ModClusterDefinition.INSTANCE.getPathElement())
+                                .addAttributes(
```

Comment:
If you convert these attributes to an enumeration, you can replace all of this with:
new AttributeXMLBuilderOperator(ModClusterDefinition.Attribute.class).apply(builder)